All Core Devs Meetings/Meeting Template.md

```diff
@@ -1,14 +1,593 @@
-# All Core Devs Meeting #
-### Meeting Date/Time: Friday xx/xx/17 at 14:00 UTC
-### Meeting Duration 2 hours
-### [Audio/Video of the meeting]()
+# All Core Devs Meeting 113 #309
+### Meeting Date/Time: Friday 05/14/2021 at 14:00 UTC
+### Meeting Duration: 1:20:17
+### [Audio/Video of the meeting](https://youtu.be/H_T2nNrTuWQ![image](https://user-images.githubusercontent.com/51566205/118755269-2ee99b00-b82e-11eb-80ba-ec5db201852d.png)
+)
 ### [Reddit thread]()
```

Comment:
What is this for ?

Reply:
This is for the Transcription of Friday, May 14th  All Core Devs #113 Meeting. 
(If I made a PR in the incorrect location please let me know)

Reply:
I meant what is the Reddit link for ? I didn't know we typically added those, and right now the link is empty.

Reply:
@timbeiko Please hold on merging to this file.

Reply:
Yeah, did not place in Reddit. I left the place intact for if someone has Reddit and able to fill the spot. As well as for someone who refers to this as an example.

Reply:
This template can be ignored as may not have been updated in a while. Please use the other [template](https://github.com/ethereum-cat-herders/PM/blob/master/Meeting_Notes_Template.md#meeting-name-ethereum-core-devs-meeting--ethereum-20-implementers-call-ethereum-cat-herders-meeting-notes)  shared with you earlier.